nx-X11/programs/Xserver/randr/randr.c

```diff
@@ -234,8 +301,15 @@ RRInit(void)
             return FALSE;
         if (!RROutputInit())
             return FALSE;
+        if (!RRProviderInit())
+            return FALSE;
         RRGeneration = serverGeneration;
     }
+#ifndef NXAGENT_SERVER
+    if (!dixRegisterPrivateKey(&rrPrivKeyRec, PRIVATE_SCREEN, 0))
+        return FALSE;
+#endif                          /* !defined(NXAGENT_SERVER) */
+
```

Comment:
This came here through https://cgit.freedesktop.org/xorg/xserver/commit/randr/randr.c?id=faeebead7bfcc78535757ca7acc1faf7554c03b7  Are you sure we do not need it for nxagent?